docs/ISSUE_TRIAGE.md

```diff
@@ -0,0 +1,201 @@
+
+> **Note:** This information is for project Maintainers, Owners, and Admins. If you are a Contributor, then you will not be able to perform most of the tasks in this topic.
+
+The core maintainers of this project are responsible for categorizing all incoming issues and delegating any critical or important issue to other maintainers. Triage provides an important way to contribute to an open source project.
+
+A triage helps resolve issues quickly by :
+
+- Ensuring the intent of an issue is clear. The issue should clearly describe the problem the end user is experiencing as well as the steps to reproduce the problem.
+- Giving a contributor the information they need before they commit to resolving an issue.
+- Streamlining the development process by identifying and closing duplicate issues.
+
+If you don't have the knowledge or time to code, consider helping with triage. The community will thank you for saving them time by spending some of yours.
+
+## 1. Find issues that need triage
+
+The easiest way to find issues that haven't been triaged is to search for issues with the `needs-triage` label.
+
+## 2. Ensure the issue contains basic information
+
+When creating a new issue, the author must provide all of the information required by the project's GitHub issue template
+
+### Standard issue information that must be included
+
+The following section describes the various issue templates and the expected content.
+
+#### Bug reports
+
+A bug report should contain explanation of what the problem is, what the expected outcome is, and how to reproduce the problem. Additionally, any other supporting information such as screenshots, console logs, and environment details should be included in the bug report. 
+
+ - Product Name/Version
+ - Component/Module Name
+ - Ansible Collection Version	
+ - Configuration
+ - Steps to Reproduce
+ - Expected/Actual Behavior
+ - Logs/Screenshots
+ - Any other additional information...
+
+#### Feature requests
+
+Should explain what feature that the author wants to be added and why that is needed.
+
+#### Ask a Question requests
+
+In general, if the issue description and title are perceived as questions no more information is needed.
+
+### Good practices
+
+To make issue handling easier for everyone, it is suggested to:
+
+- Make sure that issue titles are named to explain the subject of the issue, have correct spelling, and don't include irrelevant information and/or sensitive information.
+- Make sure that issue descriptions don't include irrelevant information.
+- Make sure that issues do not contain sensitive information.
+- Make sure that issues have all relevant fields filled in.
+- If an issue is unclear, then try to edit the title and description for more clarity or leave a comment requesting that edits to the issue be made.
+
+> **Note:** Above rules are applicable to both new and existing issues.
```

Comment:
I don't think this line is needed.

Reply:
Okay. We will discuss with server storage team and will get back to you.

Reply:
I just think that saying "rules are applicable to both new and existing issues" is already implied (why would they not be applicable?) and doesn't really add any value to the document.

Reply:
Removed as suggested.